core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -479,25 +479,33 @@ class LogManager(logDirs: Seq[File],
 
     try {
       for ((dir, dirJobs) <- jobs) {
-        dirJobs.foreach(_.get)
+        val hasErrors = dirJobs.exists {
+          future =>
```

Comment:
Nit: this should be in the previous line.

Reply:
Done.